<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pocetna</class>
 <widget class="QMainWindow" name="Pocetna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Početna | Aplikacija Vežbe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>871</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="spisak">
     <attribute name="title">
      <string>UNOS SPISKA</string>
     </attribute>
     <widget class="QLabel" name="smena">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>831</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>SMENA: PREPODNEVNA</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="brKabineta">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>80</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>50</string>
      </property>
     </widget>
     <widget class="QLabel" name="kabL">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>BROJ KABINETA*</string>
      </property>
     </widget>
     <widget class="QLabel" name="profL">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PREDMETNI PROFESOR:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="imeProf">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>150</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Danijel Jovanović</string>
      </property>
     </widget>
     <widget class="QLabel" name="fondL">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>BROJ ČASOVA</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="brojCasova">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>200</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>3</string>
      </property>
     </widget>
     <widget class="Line" name="razdvoji">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>20</width>
        <height>311</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="brCasova">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>210</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ČAS/OVA NEDELJNO, PO ODELJENJU</string>
      </property>
     </widget>
     <widget class="QLabel" name="napomena">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;*Broj kabineta se unosi samo ako dati &lt;br/&gt;profesor mora držati vežbe u tom kabinetu.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="odeljenjeL">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>RAZRED I ODELJENJE:</string>
      </property>
     </widget>
     <widget class="QLabel" name="grupaL">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>GRUPA:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="razredCb">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>260</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="odeljenje">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>260</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>5</string>
      </property>
     </widget>
     <widget class="QComboBox" name="grupaCb">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>C</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="upisUBazu">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>50</y>
        <width>51</width>
        <height>321</height>
       </rect>
      </property>
      <property name="text">
       <string>U
P
I
S

U

B
A
Z
U</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="rasporedjeno">
     <attribute name="title">
      <string>RASPORED</string>
     </attribute>
     <widget class="QComboBox" name="dan">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>PONEDELJAK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UTORAK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SREDA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ČETVRTAK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PETAK</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="prepodne">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>831</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>SMENA: PREPODNEVNA</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="prikaziRaspored">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>80</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PRIKAZ RASPOREDA</string>
      </property>
     </widget>
     <widget class="QTableView" name="prikaz">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>571</width>
        <height>281</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QGroupBox" name="rasporedGB">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>80</y>
        <width>241</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string>RASPORED</string>
      </property>
      <widget class="QPushButton" name="generisanjeBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>GENERISANJE</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>143</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QGroupBox" name="napredne">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>241</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>NAPREDNE AKCIJE</string>
       </property>
       <widget class="QPushButton" name="deleteRasporedBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>ISPRAZNI RASPORED</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteSpisakBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>ISPRAZNI SPISAK</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="napomena_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>*Raspored se generiše 
po unapred 
definisanom algoritmu.</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="oprogramu">
     <attribute name="title">
      <string>O APLIKACIJI</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>831</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(':/root/SLIKE/about.png') norepeat;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>520</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IZLAZ</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
